src/libs/ValidationUtils.js

```diff
@@ -301,6 +302,18 @@ function isValidRoutingNumber(number) {
     return false;
 }
 
+/**
+ * Returns object with character limit errors. If any are valid,
+ * those errors get returned as empty strings.
+ *
+ * @param {Number} limit
+ * @param {Object[]} toBeValidated
+ * @returns {Object[]}
+ */
+function getCharacterLimitErrors(limit, toBeValidated) {
```

Comment:
```suggestion
function doesFailCharacterLimit(maxLength, valuesToBeValidated) {
```

Reply:
What do you think about this change? We are not really getting any errors from this function. It only checks if a value is of invalid length

Reply:
Could you please also rename the parameter names so that makes more sense?